client-app/components/atoms/image-gallery/image-gallery.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div
+    ref="mainImageDiv"
+    class="square relative flex flex-col justify-center items-center border border-gray-100 rounded-sm"
+  >
+    <img
+      :src="activeSrc"
+      alt="product.name"
+      class="absolute top-0 w-full h-full object-cover object-center rounded-sm"
+    />
+  </div>
+  <div v-if="isMobile && images && images.length > 1" class="mt-4 flex flex-row justify-center space-x-2.5">
+    <div
+      v-for="image in images"
+      :key="image?.url || ''"
```

Comment:
key should be unique. It's not good to use `''` as key even as a fallback. Please, use `v-for="(image, i) in images" :key="image?.url || i"` to build as unique index as possible.